modules/update-common-terms.adoc

```diff
@@ -20,7 +20,7 @@ Recommended update edge:: A _recommended update edge_ is a recommended update be
 
 ifndef::openshift-origin[]
 
-Extended Update Support:: All post-4.7 even-numbered minor releases are labeled as _Extended Update Support_ (EUS) releases. These releases introduce a verified update path between EUS releases, permitting customers to streamline updates of worker worker nodes and formulate update strategies of EUS-to-EUS {product-title} releases that will cause fewer reboots of worker nodes.
+Extended Update Support:: All post-4.7 even-numbered minor releases are labeled as _Extended Update Support_ (EUS) releases. These releases introduce a verified update path between EUS releases, permitting customers to streamline updates of worker nodes and formulate update strategies of EUS-to-EUS {product-title} releases that will cause fewer reboots of worker nodes.
```

Comment:
```suggestion
Extended Update Support:: All post-4.7 even-numbered minor releases are labeled as _Extended Update Support_ (EUS) releases. These releases introduce a verified update path between EUS releases, permitting customers to streamline updates of worker nodes and formulate update strategies of EUS-to-EUS {product-title} releases that results in fewer reboots of worker nodes.
```

Reply:
Change looks good and one other thing to avoid future tense if possible, according to IBMSG.